projects/elife.yaml

```diff
@@ -179,67 +177,76 @@ defaults:
                                   # us-east-1, build date 20170811, hvm:ebs-ssd, AMI built on 20171215
+        # deprecated
         1404:
-            description: uses Ubuntu 14.04 (Trusty) instead of 16.04 (Xenial)
+            description: uses Ubuntu 14.04 (Trusty)
             ec2:
-                ami: ami-92002785 # Ubuntu 14.04, deprecated
+                ami: ami-92002785 # created from a 14.04 basebox instance "basebox.2016-11-03"
+                                  # possibly based on ami-9eaa1cf6 (see basebox aws-alt 14.04)
+        # 18.04
+        s1804:
+            description: isolated from the master-server and uses Ubuntu 18.04 (Bionic)
+            ec2:
+                ami: ami-00aedeb287fe8c9d6 # GENERATED created from basebox--1804
+                                           # us-east-1, build date 20180814, hvm:ebs-ssd, AMI built on 20180828
+                masterless: true
+        # 16.04, deprecated
         standalone1604:
             description: isolated from the master-server and uses Ubuntu 16.04 (Xenial)
             ec2:
                 ami: ami-14a1d06e # GENERATED created from basebox--1604
                                   # us-east-1, build date 20170811, hvm:ebs-ssd, AMI built on 20171215
                 masterless: true
+        # 16.04, deprecated
         s1604:
             description: alias for standalone1604 short enough to fit into hostnames
             ec2:
-                ami: ami-14a1d06e
+                ami: ami-14a1d06e # GENERATED created from basebox--1604
+                                  # us-east-1, build date 20170811, hvm:ebs-ssd, AMI built on 20171215
                 masterless: true
+        # 14.04, deprecated
         standalone1404:
             description: isolated from the master-server and uses (deprecated) Ubuntu 14.04 (Trusty)
             type: t2.small
             ec2:
                 ami: ami-92002785 # Ubuntu 14.04, deprecated
                 masterless: true
+        # 18.04, same as default
         standalone:
-            description: isolated from the master-server and uses Ubuntu 16.04 (Xenial)
-            ec2:
-                ami: ami-14a1d06e # GENERATED created from basebox--1604
-                                  # us-east-1, build date 20170811, hvm:ebs-ssd, AMI built on 20171215
-                masterless: true
-        standalone1804:
-            description: isolated from the master-server and uses Ubuntu 18.04 (Bionic)
-            ec2:
-                ami: ami-00aedeb287fe8c9d6 # GENERATED created from basebox--1804
-                                           # us-east-1, build date 20180814, hvm:ebs-ssd, AMI built on 20180828
-                masterless: true
-        s1804:
-            description: alias for standalone1604 short enough to fit into hostnames
+            description: isolated from the master-server and uses same AMI as default configuration
             ec2:
-                ami: ami-00aedeb287fe8c9d6
```

Comment:
went through all of the `ami` fields at https://console.aws.amazon.com/ec2/v2/home?region=us-east-1#Images:visibility=owned-by-me;search=92002785;sort=imageId and everything checks out